### PR TITLE
[supply] Added internal track

### LINKED ISSUE #12117: [supply] Add support for internal test
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description

Google introduced internal test as a new way to distribute apks to testers.
So I tried to setup supply to upload my apk to intern test track, but supply doesn't allow track which are not in [this](https://github.com/fastlane/fastlane/blob/master/supply/lib/supply/uploader.rb#L184) array.

Maybe it is enough to expand the array and add the right track name?

I couldn't find the correct track name in the [google developer api docs](https://developers.google.com/android-publisher/api-ref/edits/tracks) , but the docs was last updated on 15 February 2018.

It would be great if someone could find a way to bring internal testing support to supply :)

### PR DESCRIPTION
**Pull request by joshdholtz**
Fixes #12117

- Allowed for new `internal` track for supply

### CODE REVIEW

File: supply/spec/uploader_spec.rb
Comment (janpio) [inline supply/spec/uploader_spec.rb]: I don't understand how this code works, but that the number <-> track combination is now different doesn't mean anything?

Comment (joshdholtz) [reply supply/spec/uploader_spec.rb]: Yeah yeah, so I bumped up the versions so that `production` has highest version and `internal` has lowest version. `internal` needs to be the lowest so I just bumped up the other ones so that I could insert `internal` as the lowest.

Comment (joshdholtz) [reply supply/spec/uploader_spec.rb]: This tests tried to remove any older versions than the latest production version

**Comment by nekator** [comment]
@joshdholtz So internal is the right track name? Is it documented somewhere? I don't doubt it but I'm interested in how you figured it out.

**Comment by mandybess** [comment]
@nekator yupp 

documented here: https://developers.google.com/apis-explorer/#search/Google%20Play%20Developer%20API/androidpublisher/v2/androidpublisher.edits.tracks.update

**Comment by nekator** [comment]
@mandybess thanks for the link.

**Comment by joshdholtz** [comment]
@nekator Yup, internal is the correct link! I uploaded an APK to the internal track and verified it via Google Play dashboard 😊 

screen shot 2018-03-22 at 10 42 04 am

**Comment by nekator** [comment]
@joshdholtz thanks. 
Now I can't wait until this pr will get merged 🤩

File: supply/lib/supply/uploader.rb
Comment (minuscorp) [inline supply/lib/supply/uploader.rb]: why not `%w(production beta alpha internal rollout)`?

Comment (KrauseFx) [reply supply/lib/supply/uploader.rb]: Either way is fine, we can leave it as is

**Review by snatchev — APPROVED** [review]
LGTM

**Pull request merged**
